addons/mail/static/tests/qunit_suite_tests/components/channel_member_list_tests.js

```diff
@@ -143,5 +143,34 @@ QUnit.test('there should be a button to hide member list in the thread view topb
     );
 });
 
+QUnit.test('should show a button to load more members if they are not all loaded', async function (assert) {
```

Comment:
while we are at it it would be nice to have another test that actually clicks on the button and check that more members are actually displayed